src/main/java/frc/robot/commands/FindAndGotoNote.java

```diff
@@ -51,10 +61,15 @@ public void execute() {
 
 //            System.out.println("Note found!  ---------------");
             PhotonTrackedTarget target = getBestTarget(pipelineResult);
+            DataLogManager.log("Note Found! -------");
+
+            if (target == null) {
+                DataLogManager.log("Note Not Actually found");
```

Comment:
This should never happen .. . well, it's kind of weird because getBestTarget(pipeline) can return null, but it will never do so because it's only called if pipeline.hasTargets() is true ...

Reply:
Was just a check I though of, never actually happened